app/lib/account/backend.dart

```diff
@@ -213,10 +215,19 @@ AuthenticatedGithubAction? _tryAuthenticateGithubAction(String token) {
   if (payload == null) {
     return null;
   }
```

Comment:
I propose we do an explicit `iss` check here.

I'm not sure I mind throwing `FormatException` in `GitHubJwtPayload.tryParse` if the issuer has mismatch, but it also seems a bit wrong to throw `FormatException` due to a validation issue.

This seems like a place where an issuer check should take place, so maybe just duplicate it here, or move it here for clarity.

Reply:
Yes, my suggestion was that checking the issuer is a validation step, that should not happen in a `tryParse` function. It's all nits.

Reply:
I'll do a follow-up PR, as I also have some other refactors pending.